services/GovDeliveryService.ts

```diff
@@ -48,6 +52,53 @@ export interface PublishingSupportEmail {
   apiInternalOnlyDetails?: string;
   apiOtherInfo?: string;
 }
+export interface ContactDetails {
+  firstName: string;
+  lastName: string;
+  email: string;
+}
+export interface ProductionAccessSupportEmail {
+  primaryContact: ContactDetails;
+  secondaryContact: ContactDetails;
+  organization: string;
+  appName: string;
+  appDescription: string;
+  statusUpdateEmails: string[];
+  valueProvided: string;
+  businessModel?: string;
+  policyDocuments: string[];
+  phoneNumber: string;
+  apis?: string;
+  monitizedVeteranInformation: boolean;
+  monitizationExplanation?: string;
+  veteranFacing?: boolean;
+  website?: string;
+  signUpLink?: string;
+  supportLink?: string;
+  platforms?: string[];
+  veteranFacingDescription?: string;
+  vasiSystemName?: string;
+  credentialStorage: string;
+  storePIIOrPHI: boolean;
+  storageMethod?: string;
+  safeguards?: string;
+  breachManagementProcess?: string;
+  vulnerabilityManagement?: string;
+  exposeHealthInformationToThirdParties?: boolean;
+  thirdPartyHealthInfoDescription?: string;
+  scopesAccessRequested?: string[];
+  distrubitingAPIKeysToCustomers?: boolean;
+  namingConvention?: string;
+  centralizedBackendLog?: string;
+  listedOnMyHealthApplication?: boolean;
+}
+export interface ProductionAccessConsumerEmail {
```

Comment:
is this actually used anywhere?

Reply:
No this is dead code from when I was trying to send the images as base64. removing in next commit